subsys/usb/class/Kconfig

```diff
@@ -80,6 +80,27 @@ config MASS_STORAGE_DISK_NAME
 	help
 	  Mass storage device disk or drive name
 
+config MASS_STORAGE_INQ_VENDOR_ID
+	string "T10 assigned vendor ID for inquiry (must be 8 characters)"
+	depends on USB_MASS_STORAGE
+	default "ZEPHYR  "
+	help
+	  Vendor ID used for enquiry requests - pad with spaces to reach 8 characters
+
+config MASS_STORAGE_INQ_PRODUCT_ID
+	string "Product ID for inquiry (must be 16 characters)"
+	depends on USB_MASS_STORAGE
+	default "ZEPHYR USB DISK "
+	help
+	  Product ID used for enquiry requests - pad with spaces to reach 8 characters
```

Comment:
s/8/16/

Reply:
Might be better to clarify where the padding goes, e.g. "Spaces must be added to the end of the ID to bring the string length to 16 characters".  (Though technically it's probably "bytes", as I assume unicode identifiers are allowed.)

Reply:
damnit, fixed :)